__tests__/integration.test.js

```diff
@@ -0,0 +1,30 @@
+const app = require("../app");
+const request = require("supertest");
+const seed  = require("../db/seeds/seed")
+const db = require("../db/connection")
+const testData = require("../db/data/test-data/index")
+
+beforeEach(() => {
+    return seed(testData)
+})
+
+afterAll(()=> {
+    db.end()
+})
+
+describe('getTopics', () => {
+    it('should return a 200 status code', () => {
```

Comment:
same here - already tested in the next test. For server testing it's OK to check status and response together